README.md

```diff
@@ -8,7 +8,6 @@ HLAminer is a software for HLA predictions from next-generation shotgun (NGS) se
 
 * [hlaminer 1.4](https://github.com/bcgsc/HLAminer/releases/download/v1.4/HLAminer_1-4.tar.gz)
 * [bwa 0.7.12](https://github.com/lh3/bwa/archive/0.7.12.tar.gz)
```

Comment:
README says bwa/0.7.12 but bwa/0.7.17 is used

Reply:
Oops, good catch, thanks Peter!